examples/darwin-framework-tool/commands/pairing/PairingCommandBridge.h

```diff
@@ -22,15 +22,16 @@
 
 enum class PairingMode
 {
-    None,
+    Unpair,
     Code,
     Ble,
     AlreadyDiscoveredByIndex,
 };
 
-enum class PairingNetworkType
+enum class CommissioningType
 {
-    None,
+    None, // establish PASE only
+    NoNetwork,
```

Comment:
```suggestion
    OnNetwork,
```
I would think?

Reply:
No it's mean to mean "Don't configure the network", will rename to something more obvious

Reply:
Will address as a follow-up